DataFormats/ForwardDetId/interface/BTLDetId.h

```diff
@@ -18,20 +18,45 @@ class BTLDetId : public MTDDetId {
   
  private:
   
-  static const uint32_t kBTLmoduleOffset           = 10;
-  static const uint32_t kBTLmoduleMask             = 0x3F;
-  static const uint32_t kBTLmodTypeOffset          = 8;
-  static const uint32_t kBTLmodTypeMask            = 0x3;
-  static const uint32_t kBTLCrystalOffset          = 0;
-  static const uint32_t kBTLCrystalMask            = 0x3F;
+  static constexpr uint32_t kBTLmoduleOffset           = 10;
+  static constexpr uint32_t kBTLmoduleMask             = 0x3F;
+  static constexpr uint32_t kBTLmodTypeOffset          = 8;
+  static constexpr uint32_t kBTLmodTypeMask            = 0x3;
+  static constexpr uint32_t kBTLCrystalOffset          = 0;
+  static constexpr uint32_t kBTLCrystalMask            = 0x3F;
+
+ public:
+
+  /// range constants, need two sets for the time being (one for tiles and one for bars)
+  static constexpr int kModulesPerROD = 54;
+  static constexpr int kTypeBoundaries[4] = { 0, 18, 36, 54 };;
+  static constexpr int kCrystalsInPhiTile = 16; // per module and ROD
+  static constexpr int kCrystalsInEtaTile = 4; // per module
+  static constexpr int kCrystalsInPhiBar = 4; // per module and ROD
+  static constexpr int kCrystalsInEtaBar = 16; // per module
+  static constexpr int kCrystalsPerROD = 3456; // 64 crystals per module x 54 modules per rod
```

Comment:
can this be calculated directly as `kModulesPerROD*kCrystals...` ?

Reply:
done, Tile or Bar give the same result